.circleci/config.yml

```diff
@@ -55,6 +55,7 @@ commands:
           command: |
             . << parameters.python-environment >>/bin/activate
             make clean
+            pip install --upgrade pip
```

Comment:
was something broken because of this? could we fix the pip version in eg `requirements-dev.txt` instead?

Reply:
This was because I only tested on linux, don't know what can happen if we make this a global install - should be fine I guess but wanted to avoid the risk

Reply:
I guess it might be better to put it in the `requirements-dev.txt`

Reply:
I think `requirements-dev.txt` is better yes, that way we at least keep our dev process a bit more predictable 👍